internal/services/loganalytics/log_analytics_cluster_customer_managed_key_resource.go

```diff
@@ -274,14 +281,22 @@ func resourceLogAnalyticsClusterCustomerManagedKeyDelete(d *pluginsdk.ResourceDa
 		return fmt.Errorf("deleting `azurerm_log_analytics_cluster_customer_managed_key` %s: `customer managed key does not exist!`", *id)
 	}
 
+	// Ensure `associatedWorkspaces` is not present in request, this is a read only property and cannot be sent to the API
+	// Error: updating Customer Managed Key for Cluster
+	//		performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error:
+	//		InvalidParameter: 'properties.associatedWorkspaces' is a read only property and cannot be set.
+	//		Please refer to https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/logs-dedicated-clusters#link-a-workspace-to-the-cluster for more information on how to associate a workspace to the cluster.
```

Comment:
I'm not super fussed about this, but by this point this rather verbose comment is present three times in the resource. 

What do you think about leaving a comment referring to the comment in the create, or just a condensed version of the comment e.g.
```suggestion
	// This is a read only property, please see comment in the create function
```

or 

```
// Ensure `associatedWorkspaces` is not present in request, this is a read only property and cannot be sent to the API
```